internals/mocks/fixtures/standard-texts.json

```diff
@@ -0,0 +1,161 @@
+{
```

Comment:
Because you want to ave the same test data every time. If you make a live connection with the backend this is subject to change. And you need a working backend to run the tests.